api/swagger-spec/oapi-v1.json

```diff
@@ -18166,7 +18166,8 @@
     "required": [
      "strategy",
      "triggers",
-     "replicas"
+     "replicas",
+     "transient"
```

Comment:
No, I thought it had omit empty.  Will add the scaler.  We need to debate the name of the field.